src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -207,16 +207,25 @@ public function validate(mixed $value, Constraint $constraint): void
             }
         }
 
-        $errorPath = $constraint->errorPath ?? current($fields);
-        $invalidValue = $criteria[$errorPath] ?? $criteria[current($fields)];
+        $errorPath = $constraint->errorPath ?? implode(', ', array_keys($criteria));
```

Comment:
if one field, error stay on field, but if there are many fields, error go on top form instead one arbitrary field

Reply:
it'd be great to have a test case that covers the issue that @xabbuh spotted here

Reply:
Thinking about this PR again I'd say if we were to change anything here the constraint could accept the `errorPath` option to be configured as an array which would lead to the violation being added to all these property paths. But doing the suggested change is a no go to me as it breaks the Form integration as explained.

Reply:
add to all property can be OK but can create backward compatibility, 
but actually we have violation on only one field is not good, it's not A field  whitch is bad but a group, 
if you set firstname and lastname unique, form will respond to you "firstname is incorrect Christian is already used".
That's why put this to the parent form instead one field is preferable.
I think.
I'm trying to do tests and I'll look for an array (or an object stringable) for errorPath